apps/rekrutteringstreff-api/src/main/kotlin/no/nav/toi/jobbsoker/JobbsøkerOutboundEndpoints.kt

```diff
@@ -0,0 +1,54 @@
+package no.nav.toi.jobbsoker
+
+import io.javalin.Javalin
+import io.javalin.http.Context
+import io.javalin.http.HttpStatus
+import io.javalin.http.InternalServerErrorResponse
+import io.javalin.openapi.*
+import no.nav.toi.Rolle
+import no.nav.toi.authenticatedUser
+import no.nav.toi.kandidatsok.KandidatsøkKlient
+import java.util.*
+
+private const val pathParamPersonTreffId = "personTreffId"
+private const val endepunktRekrutteringstreff = "/api/rekrutteringstreff"
+private const val eksternKandidatnummerPath = "$endepunktRekrutteringstreff/jobbsoker/{$pathParamPersonTreffId}/kandidatnummer"
+
+data class KandidatnummerDto(val kandidatnummer: String)
+
+@OpenApi(
+    summary = "Hent kandidatnummer for en jobbsøker basert på personTreffId",
+    operationId = "hentKandidatnummerForPersonTreffId",
+    tags = ["jobbsøker-outbound"],
+    pathParams = [OpenApiParam(
+        name = pathParamPersonTreffId,
+        type = UUID::class,
+        required = true,
+        description = "Jobbsøkerens unike identifikator for treffet (UUID)"
+    )],
+    responses = [
+        OpenApiResponse("200", content = [OpenApiContent(from = KandidatnummerDto::class)]),
+        OpenApiResponse("404", description = "Jobbsøker eller kandidatnummer ikke funnet")
+    ],
+    path = eksternKandidatnummerPath,
+    methods = [HttpMethod.GET]
+)
+private fun hentKandidatnummerHandler(
+    repo: JobbsøkerRepository,
+    client: KandidatsøkKlient
+): (Context) -> Unit = { ctx ->
+    ctx.authenticatedUser().verifiserAutorisasjon(Rolle.ARBEIDSGIVER_RETTET, Rolle.UTVIKLER)
+    val personTreffId = PersonTreffId(ctx.pathParam(pathParamPersonTreffId))
+    val userToken = ctx.attribute<String>("raw_token")
+        ?: throw InternalServerErrorResponse("Raw token ikke funnet i context")
+
+    repo.hentFødselsnummer(personTreffId)?.let { fødselsnummer ->
+        client.hentKandidatnummer(fødselsnummer, userToken)?.let { kandidatnummer ->
+            ctx.json(KandidatnummerDto(kandidatnummer.asString))
+        } ?: ctx.status(HttpStatus.NOT_FOUND)
+    } ?: ctx.status(HttpStatus.NOT_FOUND)
+}
+
+fun Javalin.handleJobbsøkerOutbound(repo: JobbsøkerRepository, client: KandidatsøkKlient) {
```

Comment:
Hvorfor outbound

Reply:
tenkte det var endepunkt som går mot andre applikasjoner. Blir et par til av de, blant annet pdl, så tenkte å samle de her, men si fra om du har bedre navn på de. Eller om jeg burde ha egen endepunktslkasse for kandidatsøk kall.